### ISSUE TITLE
On the learning rate of Bert and linear classifiers in tasks

### ISSUE THREAD

**Issue by Timothyxxx** [microsoft/Unicoder#3]
Hello, reading your code run_ xnli.py During parameter adjustment, it seems that the same learning rate and the same parameter "learning_rate" are set in the optimizer of the code for the Bert and linear modules. However, in the previous research internship, we set these two learning rates at different learning rates. I'm very puzzled. Which one is right or both? I'm looking forward to your reply

**Comment by Eureka6174** [comment]
Hello, I think you could decide this based on the performance. We didn't do too much parameter tuning and just take the most straight forward solution. Just out of curious, did you got better performance with different learning rate?

**Comment by Timothyxxx** [comment]
Thank you for your answer !We do tried some combinations(both optimizers(Adam/AdamW) and learning rates) during the past few days and found that there was little difference in the results when compared to the single AdamW fixed lr (at least in our work)... but have to say there is too little information on this haha

**Comment by Eureka6174** [comment]
Thanks for your sharing, both work or not work are helpful :)
